dagmc_stats/DagmcFile.py

```diff
@@ -106,6 +109,34 @@ def __set_entityset_ranges(self):
                 self._my_moab_core.get_entities_by_type_and_tag(self.root_set, types.MBENTITYSET,
                                                                 self.dagmc_tags['geom_dim'], [dimension])
 
+    def get_meshset_by_id(self, dim, ids=[]):
+        """Get meshset of the geometry with specified dimension and ids
+
+        inputs
+        ------
+        dim : dimension of the meshset
+        ids : ids of the meshset
+
+        outputs
+        -------
+        meshset : meshset of the geometry with given dimension and ids
+        """
+        # if no id is passed in
+        if len(ids) == 0:
+            return self.entityset_ranges[dim]
+
+        all_global_id = []
+        for id in ids:
+            all_global_id.extend(list(self._my_moab_core.get_entities_by_type_and_tag(self.root_set,
+                                                                                      types.MBENTITYSET, self.dagmc_tags['global_id'], [id])))
+        meshset = list(set(self.entityset_ranges[dim]) & set(all_global_id))
+        # if id is not in the given dim range
+        if not meshset:
+            warnings.warn(
+                'ID is not in the given dimension range! All entities of specified dimension will be returned.')
```

Comment:
```suggestion
                'ID is not in the given dimension range! ' +\
                    'All entities of specified dimension will be returned.')
```
PEP8 suggestion for the line length